packages/patch-project/src/cli/dir.ts

```diff
@@ -1,9 +1,13 @@
-import fs from 'fs-extra';
+import fs from 'fs';
 
 export async function directoryExistsAsync(file: string): Promise<boolean> {
   return (await fs.promises.stat(file).catch(() => null))?.isDirectory() ?? false;
 }
 
-export const ensureDirectoryAsync = (path: string) => fs.promises.mkdir(path, { recursive: true });
+export async function ensureDirectoryAsync(path: string): Promise<void> {
+  await fs.promises.mkdir(path, { recursive: true });
+}
 
-export const moveAsync = fs.move;
+export async function moveAsync(src: string, dest: string): Promise<void> {
+  await fs.promises.rename(src, dest);
```

Comment:
Is this analogous to move?

Reply:
Alright, looked at the callsites, and there's actually a risk at two of them that we _may_ get `EXDEV` errors i.e. invalid move across devices. I've added handling to the two cases where that can happen (afaict)